packages/web-components/src/components.d.ts

```diff
@@ -2052,6 +2056,10 @@ declare namespace LocalJSX {
           * Set the input to required and render the (Required) text.
          */
         "required"?: boolean;
+        /**
+          * Whether or not the component will use USWDS v3 styling.
+         */
+        "uswds"?: boolean;
```

Comment:
Do we know why this is being duplicated here? I don't think it's a big deal, just curious.

Reply:
This file is automatically generated by the Stencil compiler but it looks like it's adding one instance to the [LocalJSX namespace](https://github.com/department-of-veterans-affairs/component-library/blob/7da4d51fd85e258f230a966745ee665e405a2624/packages/web-components/src/components.d.ts#L1386) and the other to the [Components namespace](https://github.com/department-of-veterans-affairs/component-library/blob/7da4d51fd85e258f230a966745ee665e405a2624/packages/web-components/src/components.d.ts#L8).

Which is then referenced by the [va-number-input interface](https://github.com/department-of-veterans-affairs/component-library/blob/7da4d51fd85e258f230a966745ee665e405a2624/packages/web-components/src/components.d.ts#L2628) so this is likely something to do with how the React binding for the web components work. I didn't dig into it further though.